index.js

```diff
@@ -258,6 +262,10 @@ if ((os.platform() === 'win32' && require('is-administrator')()) || process.getu
             return macaddress.all(cb);
         });
     }], (error, all) => {
+        if(error) {
```

Comment:
no need for error checking - because there might be errors in `distro` and `macaddress` but they shouldn't stop the execution.

Reply:
nope, we don't